src/gui/openfilemanager.cpp

```diff
@@ -61,12 +61,10 @@ static QString findDefaultFileManager()
         return QString();
 
     QFileInfo fi;
-    QStringList dirs = xdgDataDirs();
-    QStringList subdirs;
-    subdirs << "/applications/"
-            << "/applications/kde4/";
-    foreach (QString dir, dirs) {
-        foreach (QString subdir, subdirs) {
+    const auto &dirs = xdgDataDirs();
+    const QStringList subdirs{"/applications/", "/applications/kde4/"};
```

Comment:
Can you move the list directly to the loop.
Also when touching strings can you  pls convert them to QStringLiteral

Reply:
Done. I didn't touch strings because the PR is about detaching qt containers in range-based for loop, so I didn't want to mix the things up. If you want so I can add it, no problem.